tests/infra_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"crypto/tls"
 	"encoding/json"
```

Comment:
regarding commit message of `infra_test, Adapt tests to dual stack`:
I think using `4140` (I think you refer to the PR number right?) is a bit  ambiguous
We may not stay on Github platform forever, and  if that happens then this bumer  will loose it's meaning. 
I would avoid mentioning github PR numbering on git messages, or at least use a full url

Reply:
btw this is not the same for PR description where PR links are more than welcome to make the PR context more clear

Reply:
4140 is the test_id, its just a coincidence

Reply:
ok then please use `test_id:4140` to avoid ambiguity